Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp

```diff
@@ -624,6 +623,63 @@ void APlayCppSdkActor::SignEip155Transaction(
     });
 }
 
+void APlayCppSdkActor::SendEip155Transaction(
+    FWalletConnectTxEip155 info,
+    FWalletconnectSendEip155TransactionDelegate Out) {
+    ::com::crypto::game_sdk::WalletconnectClient *coreclient = GetClient();
+    // if no walletconnect session, return
+    if (coreclient == nullptr)
+        return;
+    TArray<uint8> address = GetAddress();
+    int64 chain_id = (uint64)GetChainId();
+    // if no address, return
+    if (address.Num() == 0 || chain_id == 0)
+        return;
```

Comment:
cannot check error
need to return false via delegate

Reply:
consider, move to inside asynctask
and using runtime_error?

Reply:
It is not checking the error here, but checking if the wallet is connected or not. Before the checking, we check the wallet is connected or not via  `coreclient = nullptr`, which is not enough. GetAddress and GetChainId will return 0 if the wallet is not connected, then it will directly return.

Reply:
i see, how about return 
- Success (bool)
- Result (FString) <- error message?

then user side code can handle with just path , from delegate
because this operation is jsonrpc, will take some time to complete

Reply:
ok

Reply:
so return value is for whether user's tx is posted to "txpool" or not

Reply:
The return value is transaction hash, it is similar as https://github.com/cronos-labs/play-unreal-plugin/blob/main/Source/CronosPlayUnreal/Private/PlayCppSdkActor.cpp#L569. It either return transaction hash or timeout error (60 seconds), no need to return additional value to indicate success or not.